File: Nithya/Week-1/Plusone.java
class Solution {
    public int[] plusOne(int[] digits) {
        int total=digits.length;
        for(int i=total-1;i>=0;i++)
        {
            if(digits[i]<9)
            {
                digits[i]++;
                return digits;
            }
            digits[i]=0;
        }
        int otherwise[]=new int[total+1];
        otherwise[0]=1;
        return otherwise;
    }
}

File: Revanth/week-1/Day1/plusOne.py
#plusOne

class Solution(object):
    def plusOne(self, digits):
        l = len(digits) - 1
        while l > -1:
            if digits[l] < 9:
                digits[l] += 1
                return digits
            else:
                digits[l] = 0
                l -= 1
        return [1] + digits 